hokusai/staging.yml

```diff
@@ -139,3 +139,75 @@ spec:
                     operator: In
                     values:
                     - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-cleanup-pods-by-name
+spec:
+  schedule: "0 * * * *"
```

Comment:
Jenkins job config shows the next job run's date and time. Can copy the hour/minute from there.
<img width="924" alt="Screen Shot 2023-05-17 at 5 16 44 PM" src="https://github.com/artsy/opstools/assets/63004951/a6900e5c-8fc2-49f8-8756-59efa8edc6f0">

The nice thing about Jenkins' hour/minute is that it's randomized (the `H`). This helps to reduce the number of jobs running at the same time.

Reply:
[Kubernetes](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) does not support the `H` feature.

Reply:
We should avoid having many crons scheduled for the same moment. The `32 * * * *` schedule used by Jenkins so far seem OK.

Reply:
I didn't fully understand if `32 * * * *` was a suggestion to set that for _only_ the jenkins crons or for the `hokusai-run` ones as well. So seeing as we removed the jenkins crons I decided to set [`30 * * * *`](https://github.com/artsy/opstools/pull/29/commits/d0a978b9d93d150c5de37eb57ea83cd7030829c8#diff-8ba030368b0c3dfd60ec6bedbbb859e1ca856d64344e6e72b59ffeac4d310d3fR113) for the `hokusai-run` cleanup crons.